library/psa_crypto_ffdh.c

```diff
@@ -256,7 +209,91 @@ psa_status_t mbedtls_psa_ffdh_generate_key(
 
     return status;
 }
+
+psa_status_t mbedtls_psa_ffdh_import_key(
+    const psa_key_attributes_t *attributes,
+    const uint8_t *data, size_t data_length,
+    uint8_t *key_buffer, size_t key_buffer_size,
+    size_t *key_buffer_length, size_t *bits)
+{
+    (void) attributes;
+
+    if (key_buffer_size < data_length) {
+        return PSA_ERROR_BUFFER_TOO_SMALL;
+    }
+    memcpy(key_buffer, data, data_length);
+    *key_buffer_length = data_length;
+    *bits = PSA_BYTES_TO_BITS(data_length);
+
+    return PSA_SUCCESS;
+}
+
 #endif /* MBEDTLS_PSA_BUILTIN_KEY_TYPE_DH_KEY_PAIR ||
           MBEDTLS_PSA_BUILTIN_KEY_TYPE_DH_PUBLIC_KEY */
 
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_FFDH)
+psa_status_t mbedtls_psa_key_agreement_ffdh(
```

Comment:
Minor, can be left for later: similarly do Valerio's remark perhaps this should be called `mbedtls_psa_ffdh_key_agreement()` instead?

Reply:
Function names have been adapted here: https://github.com/Mbed-TLS/mbedtls/pull/7627/commits/a877e2bead0263e86788314f5dd2db6854455f97